source/write/transaction.txt

```diff
@@ -0,0 +1,183 @@
+.. _php-transactions:
+
+=====================
+Perform a Transaction
+=====================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code example, ACID compliance, multi-document
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+php-library+} to perform
+**transactions**. Transactions allow you to perform a series of operations
+that change data only if the entire transaction is committed.
+If any operation in the transaction does not succeed, the library stops the
+transaction and discards all data changes before they ever become
+visible. This feature is called **atomicity**.
+
+In MongoDB, transactions run within logical sessions. A
+session is a grouping of related read or write operations that you
+want to run sequentially. Sessions enable causal consistency for a group
+of operations and allow you to run operations in an **ACID-compliant**
+transaction, which is a transaction that meets an expectation of
+atomicity, consistency, isolation, and durability. MongoDB guarantees
+that the data involved in your transaction operations remains
+consistent, even if the operations encounter unexpected errors.
+
+When using the {+php-library+}, you can create a new session from a
+``MongoDB\Client`` instance. Then, you can use the resulting
+``MongoDB\Driver\Session`` instance to perform transactions. You can
+improve your app's performance by reusing your client for multiple
+sessions and transactions instead of instantiating a new client each
+time.
+
+.. warning::
+
+   Use a ``Session`` only in operations running on the
+   ``Client`` that created it. Using a ``Session`` with a
+   different ``Client`` results in operation errors.
+
+Methods
+-------
+
+Create a ``Session`` by using the ``MongoDB\Client::startSession()``
+method on your ``Client`` instance. The {+php-library+} provides a
+Convenient Transaction API to manage the transaction lifecyle. Use the
```

Comment:
I noticed the name is emphasized in the Node docs: https://www.mongodb.com/docs/drivers/node/current/usage-examples/transaction-conv/

Should that be done here?